src/org/jetbrains/java/decompiler/modules/decompiler/exps/ExitExprent.java

```diff
@@ -83,24 +82,21 @@ public TextBuffer toJava(int indent, BytecodeMappingTracer tracer) {
     tracer.addMapping(bytecode);
 
     if (exitType == EXIT_RETURN) {
-      TextBuffer buffer = new TextBuffer();
+      TextBuffer buffer = new TextBuffer("return");
 
       if (retType.type != CodeConstants.TYPE_VOID) {
-        buffer.append(" ");
-        if (indent > 450) {
```

Comment:
I think this change should stay. This prevents slowdowns on some edge cases for the decompiler.